src/tests/nativeaot/SmokeTests/Preinitialization/Preinitialization.cs

```diff
@@ -1682,6 +1685,12 @@ public static unsafe class IUnknownImpl
         [FixedAddressValueType]
         public static readonly IUnknownVftbl Vtbl;
 
+        public static nint AbiToProjectionVftablePtr
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            get => (nint)Unsafe.AsPointer(ref Unsafe.AsRef(in Vtbl));
+        }
+
         static IUnknownImpl()
         {
             ComWrappers.GetIUnknownImpl(
```

Comment:
Copy-paste error in the `IUnknownImpl` cctor, should be doing `IUnknownVftb*` casts, not `IInspectableVftbl*`?

Reply:
Thanks, opened #114422 with the fix. It shouldn't really matter.